sr_unix/ydb_env_unset.gtc

```diff
@@ -22,5 +22,4 @@ if [ -x "$ydb_dist/mumps" ] ; then
 	. $ydb_tmp_env/out
 	rm -rf $ydb_tmp_env
     fi
-else echo '$ydb_dist/mumps' executable not found
```

Comment:
I see this error is removed. But can we also do the unset in case the mumps way is not possible. That way we have a clean environment just in case a subset of the environment variables are still set at the time of the ydb_env_unset call.